src/data/reducers/assets.js

```diff
@@ -32,8 +65,14 @@ const status = (state = {}, action) => {
         response: action.response,
         type: action.type,
       };
-    case assetActions.ASSET_XHR_FAILURE:
+    case assetActions.DELETE_ASSET_FAILURE:
```

Comment:
This is sort of what makes me think that maybe a generic XHRFailure would be useful, because the only difference between the DELETE_ASSET_FAILURE and TOGGLING_LOCK_ASSET_FAILURE are that the TOGGLING_LOCK_ASSET_FAILURE gets the asset it's trying to lock (I'd argue that DELETE_ASSET_FAILURE could use that too). So generalizing it to be any ASSET_XHR_FAILURE would reduce the redundancy here, perhaps.